Library/Homebrew/test/formula_auditor_spec.rb

```diff
@@ -714,6 +714,78 @@ class Foo < Formula
       expect(fa.problems).to be_empty
     end
 
+    it "requires `branch:` to be specified for Git head URLs" do
+      fa = formula_auditor "foo", <<~RUBY, online: true
+        class Foo < Formula
+          url "https://brew.sh/foo-1.0.tgz"
+          sha256 "31cccfc6630528db1c8e3a06f6decf2a370060b982841cfab2b8677400a5092e"
+          head "https://github.com/Homebrew/homebrew-test-bot.git"
+        end
+      RUBY
+
+      fa.audit_specs
+      # This is `.last` because the first problem is the unreachable stable URL.
+      expect(fa.problems.last[:message]).to match("Git `head` URL must specify a branch name")
```

Comment:
`#last` feels a bit fragile:
```suggestion
      # This is `.last` because the first problem is the unreachable stable URL.
      expect(fa.problems.map { _1[:message] }).to include("Git `head` URL must specify a branch name")
```

Reply:
Seems fine as-is to me if the test code is functioning as expected.

Reply:
I stuck with `.last` because the others were `.first` and that seemed simplest.